style/style.css

```diff
@@ -82,29 +87,201 @@ h2 {
   background-image: url(../assets/background.png);
   background-repeat: no-repeat;
   background-size: 375px 668px;
-  padding-left: 10px;
+  background-color: rgb(255, 255, 255);
   width: 375px;
   height: 718px;
+  margin: 0 auto;
+}
+
+.content ul,
+li {
+  list-style: none;
+  display: inline;
+  padding: 1px 13px;
+  margin: -3px;
 }
 
-ul,
+.contact ul,
 li {
   list-style: none;
   display: inline;
   padding: 1px 13px;
-  margin-left: -3px;
+  margin: -3px;
+}
+
+.works h2 a {
+  color: #396df2;
+  border: 2px solid #396df2;
+  border-radius: 8px;
+  padding: 10px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+.contact h2 a {
+  color: #396df2;
+  border: 2px solid #396df2;
+  border-radius: 8px;
+  padding: 10px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  margin: 16px;
+}
+
+.contact h2 a:hover {
+  background-color: #6070ff;
+  color: rgb(255, 255, 255);
+  padding: 10px;
+  border-radius: 8px;
+}
+
+.contact h2 a:active {
+  background-color: #2230d2;
+  border: 2px solid #2230d2;
+  color: rgb(255, 255, 255);
+  padding: 10px;
+  border-radius: 8px;
+}
+
+.works h2 a:hover {
+  background-color: #6070ff;
+  color: rgb(255, 255, 255);
+  padding: 10px;
+  border-radius: 8px;
 }
 
-h2 a:hover {
+.works h2 a:active {
+  background-color: #2230d2;
+  border: 2px solid #2230d2;
+  color: rgb(255, 255, 255);
+  padding: 10px;
+  border-radius: 8px;
+}
+
+#connect a:hover {
   background-color: #6070ff;
   color: rgb(255, 255, 255);
   padding: 5px;
   border-radius: 8px;
 }
 
-h2 a:active {
+#connect a:active {
   background-color: #2230d2;
   color: rgb(255, 255, 255);
   padding: 5px;
   border-radius: 8px;
 }
+
+.big {
+  height: 596px !important;
```

Comment:
_Please look at the comment about the `!important` above._

- [ ] **OPTIONAL** In this case you're not overusing it, and it does actually forcing the height at the sections with "big" `class`. But, again! It's not the only solution, did you try to make it `min-height` instead of `height`. Give it a try and tell me if it works. 😄

Reply:
Thank you for the tip! It works pretty well!

Reply:
That's awesome 👍🏻